README.md

```diff
@@ -2,11 +2,59 @@
 title: Firebolt OpenRPC Tools
 ---
 # Firebolt OpenRPC Tools
-Tools to enable consistent Firebolt SDK generation.
+Tools to enable consistent Firebolt SDK and document generation.
 
-## Usage
-To install the package, run:
+## To Use
 
-```
-npm install @firebolt-js/openrpc
-```
\ No newline at end of file
+  - First, `npm install`.
+  - Next, invoke the cli directly with `src/cli.js`. See below for arguments.
+
+Alternatively, you may install globally using `npm install -g @firebolt-js/openrpc`, which will install the cli and add it to your $PATH so it may be invoked with the `firebolt-openrpc` command. However, trying this at the time of writing, it did not work.
```

Comment:
Any thoughts on what didn't work?

I tried to model it after standard CLI patterns I found, but I've only been using it via other npm packages, so true CLI wasn't needed. Maybe we should create an issue, and remove this sentence till it's fixed?

Reply:
Good call. Trying again, I'm not totally unsure it wasn't user error on my part. I'll remove the sentence and investigate further. Will create an issue if needed.